hermes-consumers/src/main/java/pl/allegro/tech/hermes/consumers/supervisor/process/Signal.java

```diff
@@ -57,10 +74,11 @@ boolean canExecuteNow(long currentTimestamp) {
 
     @Override
     public String toString() {
-        return "Signal{" +
-                "type=" + type +
-                ", target=" + target +
-                '}';
+        return "Signal(" + id + ", " + type +", " + target + ")";
+    }
+
+    public String getLogWithIdAndType() {
```

Comment:
I didn't want to duplicate target information in logs. target == subscriptionName so we could see something like this:

```
Process for subscription group.topic$sub no longer exists. [Signal(1, KILL, group.topic$sub)]
```